src/scriptengine/ScriptEngine.hpp

```diff
@@ -77,8 +77,12 @@ class ScriptEngine
   // issue for the underlying ScriptObject (and VALUE or PyObject), so just return the ScriptObject
   virtual ScriptObject loadMeasure(const openstudio::path& measureScriptPath, std::string_view className) = 0;
 
+  // Returns number of arguments for methodName of the object methodObject
   virtual int numberOfArguments(ScriptObject& methodObject, std::string_view methodName) = 0;
 
+  // Check if methodObject has a method called methodName. If overriden_only, will only look into the current subclass, not the parent ones
+  virtual bool hasMethod(ScriptObject& methodObject, std::string_view methodName, bool overriden_only = true) = 0;
```

Comment:
I really thought this would be needed to run older measures. And it was by far the hardest thing to achieve.
Only to realize that the base class having it makes it safe to call it even if not implemented/overidden in the actual measure...

Reply:
But the check remains? Seems like a switch statement on has "modelOutputRequests" is unneessary. Not a big deal in any case.